tests/README.md

```diff
@@ -423,7 +423,7 @@ the test driver machine.
 
 * Start by making sure you're up to date, and install git and ducktape:
 
-        $ sudo apt-get update && sudo apt-get -y upgrade && sudo apt-get install -y git
+        $ sudo apt-get update && sudo apt-get -y upgrade && sudo apt-get install -y pip git
```

Comment:
Is `pip` a package on some Debians? On Ubuntu I'm pretty sure it's called `python-pip`.